static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -454,6 +454,8 @@ define([
             if (mobileAdSizes && mobileAdSizes.indexOf(adSizes.stickyMpu) > -1) {
                 stickyMpu($adSlot);
             }
+        } else if ($adSlot.hasClass('ad-slot--facebook')) {
+            $adSlot.addClass('ad-slot--fluid');
```

Comment:
That's the problem, though - if the elements are added to the DOM with `ad-slot--facebook`, they won't be full width on their initial render. I'm not sure if users will be able to perceive this in practice, though.